### PR TITLE
[9.x] feature: `max_digits` and `min_digits` validation translations

### PR DESCRIPTION
**Pull request by danharrin**
laravel/framework#43797

### CODE REVIEW

File: lang/en/validation.php [hunk 1]
```diff
@@ -90,6 +90,7 @@
         'numeric' => 'The :attribute must not be greater than :max.',
         'string' => 'The :attribute must not be greater than :max characters.',
     ],
+    'max_digits' => 'The :attribute must not have more than :max digits.',
```
Comment (zepfietje) [inline lang/en/validation.php:93]: To match the `max.string` and `digits` rules:
```suggestion
    'max_digits' => 'The :attribute must not be greater than :max digits.',
```

File: lang/en/validation.php [hunk 2]
```diff
@@ -98,6 +99,7 @@
         'numeric' => 'The :attribute must be at least :min.',
         'string' => 'The :attribute must be at least :min characters.',
     ],
+    'min_digits' => 'The :attribute must not have less than :min digits.',
```
Comment (zepfietje) [inline lang/en/validation.php:102]:
```suggestion
    'min_digits' => 'The :attribute must be at least :min digits.',
```

File: lang/en/validation.php (hunk 1 above)
Comment (danharrin) [reply lang/en/validation.php:93]: "be greater" in a numeric context suggests the value of the number, not its length

Comment (zepfietje) [reply lang/en/validation.php:93]: Yeah already thought so, probably better as is then.

**Pull request merged**